rosa-hypershift/rosa-hosted-wrapper.py

```diff
@@ -34,6 +35,16 @@
 from random import randrange
 
 
+def set_force_terminate(signum, frame):
+    logging.warning("Captured Ctrl-C, sending exit event to watcher, any cluster install/delete will continue it execution")
```

Comment:
"will continue its execution"

Reply:
Fixed, and also deleted the duplicated check of the --workers parameter